include/vctr/Containers/VctrBase.h

```diff
@@ -69,11 +69,30 @@ class VctrBase : private Config,
     /** Returns the number of elements. This overload is a static function, used in case the extent is static. */
     static constexpr size_t size() requires (extent != std::dynamic_extent) { return extent; }
 
+    /** Returns an iterator to the begin of the storage */
     auto begin()       { return storage.begin(); }
+
+    /** Returns a const iterator to the begin of the storage */
     auto begin() const { return storage.begin(); }
+
+    /** Returns an iterator to the first element behind the storage */
     auto end()         { return is::stdArray<StorageType> ? storage.begin() + extent : storage.end(); }
```

Comment:
Well, the ternary operator was allowed in a constexpr context since the beginning of constexpr afaik. This here is actually not like an `if constexpr` thing where both branches could return a different type, it's more like we can be sure that the compiler will figure out that the condition is compile time constant and so no branching will actually happen, so this whole thing looks a lot more compact

Reply:
But your comment reminded me that these member functions are missing a constexpr keyword. Will add them...

Reply:
Aha, I somehow remember that ternary op did not work in `constexpr` (unlike `if constexpr`) and that I also found a hint somewhere on stackxc, but maybe I am mixing up things.